### PR TITLE
Prioritize patient specificity scorers

### PR DESCRIPTION
**Pull request by danielpgross**

### CODE REVIEW

File: components/specificity-meter/api/src/main/java/org/phenotips/data/internal/MonarchPatientScorer.java [hunk 1]
```diff
@@ -165,6 +165,10 @@ public double getScore(Patient patient)
         return -1;
     }
 
+    public int getScorerPriority() {
+        return 2;
```
Comment (sdumitriu) [inline components/specificity-meter/api/src/main/java/org/phenotips/data/internal/MonarchPatientScorer.java]: I would space these out, to allow something to come between monarch and omimInformationContent. Let's say 50 for this one, 10 for omimInformationContent.

File: components/specificity-meter/api/src/main/java/org/phenotips/data/internal/PatientScorerComparator.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package org.phenotips.data.internal;
```
Comment (sdumitriu) [inline components/specificity-meter/api/src/main/java/org/phenotips/data/internal/PatientScorerComparator.java]: Missing license header.

File: components/specificity-meter/api/src/main/java/org/phenotips/data/PatientScorer.java [hunk 3]
```diff
@@ -45,4 +45,11 @@
      * @return a score between {@code 0} and {@code 1}, or {@code -1} if the score cannot be computed by this scorer
      */
     double getScore(Patient patient);
+
+    /**
+     * Get the priority of this scorer. Higher priority (larger integer) scorers will be used first.
+     *
+     * @return a score greater than {@code 0}
```
Comment (sdumitriu) [inline components/specificity-meter/api/src/main/java/org/phenotips/data/PatientScorer.java]: Since this is a new method, it should have a `@since`.

**Comment by danielpgross** [comment]
@sdumitriu are you still planning to fix the test issues here?

**Comment by sdumitriu** [comment]
Yes, and I know how, I just didn't get around to it.

Or, if you want to do it yourself, see [the changes in this file](https://github.com/phenotips/phenotips/commit/9788856895395b457835b757e6de8ead837fc00c#diff-fd854e7f4b49f87fb844f05f0c4efb1a).

**Comment by danielpgross** [comment]
Ok, thanks. In that case, I'll leave it to you -- I just wanted to make sure we're on the same page.

**Comment by danielpgross** [comment]
This thread is no longer being worked on.

**Pull request closed without merging**